examples/nidcpower/measure-record.py

```diff
@@ -103,23 +95,20 @@ def check_for_initialization_error(response):
             attribute_value=nidcpower_types.NiDCPowerInt32AttributeValues.NIDCPOWER_INT32_MEASURE_WHEN_VAL_AUTOMATICALLY_AFTER_SOURCE_COMPLETE,
         )
     )
-    check_for_error(vi, configure_measure_when.status)
 
-    # set the voltage level.
+    # Set the voltage level.
     configure_voltage_level = client.ConfigureVoltageLevel(
         nidcpower_types.ConfigureVoltageLevelRequest(vi=vi, level=VOLTAGE_LEVEL)
     )
-    check_for_error(vi, configure_voltage_level.status)
```

Comment:
I'm not familiar enough with the API, but should we call the new `check_for_warning` in all the places we had `check_for_error` before? It might be the case that we don't expect warnings from these calls but I'm leaning towards keeping them to notify the user in case they do occur.

Reply:
It was important to do on each call when that was the only way to discover that an error happened. I think it just litters the examples unnecessarily to check for warnings on each call. I'd prefer to keep it limited to the places where warnings can actually happen. @bkeryan @marcoskirsch @harsha-bhushan thoughts?

Reply:
> I think it just litters the examples unnecessarily to check for warnings on each call. 

I agree.

> I'd prefer to keep it limited to the places where warnings can actually happen. @bkeryan @marcoskirsch @harsha-bhushan thoughts?

The only NI-DCPower warnings I'm aware of are measurement-related, so I would expect them to be returned from `FetchMultiple` or *maybe* `Initiate`.

Reply:
Generally speaking MI APIs don't return warnings other than to indicate things like "an overrange occurred".
In `nimi-python` APIs the warnings get logged. [See here](https://github.com/ni/nimi-python/blob/3e4372d32409594d6968c9a392b3d0078ba7e8bf/generated/nidcpower/nidcpower/errors.py#L112).

Reply:
> Generally speaking MI APIs don't return warnings other than to indicate things like "an overrange occurred".

@marcoskirsch That's what I meant by "measurement-related". Do you expect these to be returned from any function other than fetch/measure?

Reply:
nope, I was adding a but more info to your correct answer.

Reply:
I had put it on Initialize knowing that IVI defines 2 warnings for not-supported (ID query and self-test) but it doesn't seem any NI drivers use that. If someone can confirm we don't warn on those anywhere, I'll remove the first warning check.